lib/near.js

```diff
@@ -31,7 +31,7 @@ class Near {
         this.connection = connection_1.Connection.fromConfig({
             networkId: config.networkId,
             provider: { type: 'JsonRpcProvider', args: { url: config.nodeUrl, headers: config.headers } },
-            signer: config.signer || { type: 'InMemorySigner', keyStore: config.keyStore || config.deps.keyStore }
+            signer: config.signer || { type: 'InMemorySigner', keyStore: config.keyStore || (config.deps && config.deps.keyStore) }
```

Comment:
Wouldn't `keyStore` evaluate to `false` if `config.deps` is false? Or is there another check later to see if `config.keyStore` is a valid object?

Reply:
So... if `config.keyStore` exists it will be returned. If it does not exist - `config.deps.keyStore` will be returned. Or `undefined`, if they are both `undefined`.
Try to play with this example to understand it better:
```javascript
let var1;
let var2 = {"test": {"test2": "test3"}};
let testValue = var1 || (var2.test && var2.test.test2);
console.log(testValue);
```

Reply:
Got it, yeah I was just wondering if we handled the undefined case properly, but it looks like we do 👍